client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -0,0 +1,107 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useSiteSetupFlowProgress } from '../hooks/use-site-setup-flow-progress';
+import { useSiteSlugParam } from '../hooks/use-site-slug-param';
+import { USER_STORE, ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import ProcessingStep, { ProcessingResult } from './internals/steps-repository/processing-step';
+import SiteCreationStep from './internals/steps-repository/site-creation-step';
+import {
+	AssertConditionResult,
+	AssertConditionState,
+	Flow,
+	ProvidedDependencies,
+} from './internals/types';
```

Comment:
I think we should split out the type imports so we have a clear declaration of what we're importing.
```suggestion
import { AssertConditionState } from './internals/types';
import type {
	AssertConditionResult,
	Flow,
	ProvidedDependencies,
} from './internals/types';
```